bin/ripple/ecdsa/__init__.py

```diff
@@ -0,0 +1,14 @@
+__all__ = ["curves", "der", "ecdsa", "ellipticcurve", "keys", "numbertheory",
+           "test_pyecdsa", "util", "six"]
+from .keys import SigningKey, VerifyingKey, BadSignatureError, BadDigestError
+from .curves import NIST192p, NIST224p, NIST256p, NIST384p, NIST521p, SECP256k1
+
+_hush_pyflakes = [SigningKey, VerifyingKey, BadSignatureError, BadDigestError,
+                  NIST192p, NIST224p, NIST256p, NIST384p, NIST521p, SECP256k1]
+del _hush_pyflakes
+
+# This code comes from http://github.com/warner/python-ecdsa
+
+from ._version import get_versions
+__version__ = get_versions()['version']
+del get_versions
```

Comment:
That's a bit obsessive - though I've realize that this is some external code so I shouldn't critique it too much.

The pythonic way, if you're worried about exposing global variables (which is not terrible, though most people don't bother, on the theory that people simply shouldn't use stuff they aren't told to use): `from ._version import get_versions as _get_versions`